docs/pages/tutorials/secrets-tutorial.md

```diff
@@ -0,0 +1,281 @@
+
+
+## Create Secrets
+
+
+Use DCOS CLI to create a secret.  You must have the Enterprise DC/OS CLI installed.
+
+Install DC/OS Enterprise CLI:
+
+```
+$ dcos package install --cli dcos-enterprise-cli
+```
+
+Create a secret with path `hello-world/secret1`:
+
+```
+$ dcos security secrets  create -v "the value of secret1" hello-world/secret1
+```
+
+Now, lets create a key and add its private key to a secret with path `hello-world/secret2` and its public key to another secret with path `hello-world/secret3`.
+
+Create key and cert:
+
+```
+$openssl genrsa -out privatekey.pem 1024
+$openssl req -new -x509 -key privatekey.pem -out publickey.cer -days 1825
+```
+
+Create secrets with value from these private and public key files:
+
+```
+$ dcos security secrets  create -f privatekey.pem  hello-world/secret2
+```
+```
+$ dcos security secrets  create -f publickey.cer  hello-world/secret3
+```
+
+
+You can list secrets and view the content of a secret as follow:
+
+```
+$ dcos security secrets list  hello-world
+```
+```
+$ dcos security secrets get hello-world/secret3
+```
+
+
+## Install Service with Secrets
+
+
+
+The `hello-world` package includes a sample YAML file for secrets. Please examine the following `examples/secrets.yml` file to grasp how secrets are declared:
+
+```
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{SERVICE_PRINCIPAL}}
+  user: {{SERVICE_USER}}
+pods:
+  hello:
+    container:
+      image-name: ubuntu:14.04
+    count: {{HELLO_COUNT}}
+    placement: {{HELLO_PLACEMENT}}
+    secrets:
+      s1:
+        secret: {{HELLO_SECRET1}}
+        env-key: HELLO_SECRET1_ENV
+        file: HELLO_SECRET1_FILE
+      s2:
+        secret: {{HELLO_SECRET2}}
+        file: HELLO_SECRET2_FILE
+    tasks:
+      server:
+        goal: RUNNING
+        cmd: >
+               env &&
+               ls -la &&
+               cat HELLO_SECRET*_FILE &&
+        ................
+  world:
+    count: {{WORLD_COUNT}}
+    placement: {{WORLD_PLACEMENT}}
+    secrets:
+      s1:
+        secret: {{WORLD_SECRET1}}
+        env-key: WORLD_SECRET1_ENV
+      s2:
+        secret: {{WORLD_SECRET2}}
+        file: WORLD_SECRET2_FILE
+      s3:
+        secret: {{WORLD_SECRET3}}
+    tasks:
+      server:
+        goal: RUNNING
+        cmd: >
+               env &&
+               ls -la &&
+               cat WORLD_SECRET*_FILE &&
+        ................
+```
+
+
+The `hello` pod has two secrets. The first secret with path `hello-world/secret1` is exposed both as an environment variable and as a file. The second one is exposed only as a file. The value of the second secret with path `hello-world/secret2`, that is the private key, will be copied to the `HELLO_SECRET2_FILE` file located in the sandbox. 
```

Comment:
The first secret, with path `hello-world/secret1`,

, which is the private key,